lib/Standalone/ConvertTppToXsmm.cpp

```diff
@@ -115,13 +115,13 @@ struct ConvertTppBrgemmOp : public OpRewritePattern<BrgemmOp> {
     if (failed(ldaDim))
       return failure();
     int64_t lda = *ldaDim;
-    if (memrefA.getElementType().isBF16() && memrefA.getShape().size() == 3) {
-      auto divLdaDim = getLeadingDim(memrefA, 1);
+    if (memrefA.getElementType().isBF16() && memrefA.getShape().size() == 4) {
+      auto divLdaDim = getLeadingDim(memrefA, 2);
       if (failed(divLdaDim))
         return failure();
       lda = lda / (*divLdaDim);
+      b = b * memrefA.getShape()[3];
```

Comment:
what is `b` here?

Reply:
b corresponds to batch size. Instead of initializing b and updating it later, I will initialize b to take the 0th dimension of memrefB, much cleaner.

Reply:
I would spell it out. `b` is too generic